homework-submission/week5-class16/src/components/TodoList.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import Todo from "./Todo";
+import AddForm from "./AddForm";
+import { observer, inject } from "mobx-react";
+
+@inject("TodosStore")
+@observer
+class TodoList extends Component {
+  componentDidMount() {
+    this.props.TodosStore.getTodos();
+  }
+  render() {
+    const { TodosStore } = this.props;
+    const list = TodosStore.Data;
+    const todos = list.map((todo, i) => {
+      return <Todo key={i} todo={todo} />;
+    });
+
+    const rendering = list.length ? (
+      todos
+    ) : (
+      <li className="zero_todo"> The List IS Empty. Please add new todo....</li>
+    );
+
+    const loading = () => {
```

Comment:
<h1 align="center">🏅</h1>

 - But as a suggestion:
```suggestion
    const IsLoading = () => {
```

And now you can use it as a component in line 47, so instead of:
```JSX
<ul className="todo_list">{loading()}</ul>
```
It will be:
```JSX
<ul className="todo_list">
  <IsLoading />
</ul>
```